packages-ts/gauntlet-terra-contracts/src/lib/inspection.ts

```diff
@@ -1,5 +1,117 @@
 import { Proto } from '@chainlink/gauntlet-core/dist/crypto'
 import { BN } from '@chainlink/gauntlet-core/dist/utils'
+import { AccAddress, LCDClient } from '@terra-money/terra.js'
+import { providerUtils } from '@chainlink/gauntlet-terra'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+import { deepCopy } from './utils'
+import Long from 'long'
+
+// TODO: find the right place for this function
+export const getLatestOCRConfig = async (provider: LCDClient, contract: AccAddress) => {
+  // The contract only stores the block where the config was accepted. The tx log contains the config
+  const latestConfigDetails: any = await provider.wasm.contractQuery(contract, 'latest_config_details' as any)
+  const setConfigTx = providerUtils.filterTxsByEvent(
+    await providerUtils.getBlockTxs(provider, latestConfigDetails.block_number),
+    'wasm-set_config',
+  )
+
+  return setConfigTx?.logs?.[0].eventsByType['wasm-set_config']
+}
+
+enum DIFF_PROPERTY_COLOR {
+  ADDED = 'green',
+  REMOVED = 'red',
+  NO_CHANGE = 'reset',
+}
+
+type DIFF_OPTIONS = {
+  initialIndent?: string
+  propertyName?: string
+}
+
+export function printDiff(existing: Object, incoming: Object, options?: DIFF_OPTIONS) {
```

Comment:
This function should be moved to `gauntlet-core` and exposed/used on Solana also (+EVM). Please add a GH Issue on this, and add a TODO lining to the task to document.

Reply:
I'll be working on Solana side this week to adapt the same changes (unless something more urgent pops up), this is where this will migrate to gauntlet-core. 
`TODO lining to the task to document` - you mean add TODO with the issue reference to the code?

Reply:
I meant open a GH issue and add to code `// TODO: <link_to_issue>`

Reply:
Created an issue here - https://github.com/smartcontractkit/chainlink-terra/issues/181